.claude/plugins/@sea/sea-domain-forge/commands/case.md

```
@@ -0,0 +1,74 @@
+  Open, view, update, or report on a CMMN case using the PM-Agent Conductor. Supports
+  creating new cases, checking case status, advancing tasks, updating stages, and
+  generating progress communications. Use: /sea-domain-forge:case [action] [case-id?]
+---
+
+# /sea-domain-forge:case
+
+Manage CMMN cases using the PM-Agent Conductor.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)